src/gbt.cpp

```diff
@@ -85,124 +65,88 @@ bool scanGBTPhasesLocal(localArgs *la, const uint32_t ohN, const uint32_t N, con
+        EMIT_RPC_ERROR(stdsprintf("The vfatN parameter supplied (%u) exceeds the number of VFAT's per OH (%u).", vfatN, oh::VFATS_PER_OH), true);
 
     // phase check
-    if (gbt::checkPhase(la->response, phase))
+    if (checkPhase(phase))
         return true;
 
     // Write the triplicated phase registers
-    const uint32_t gbtN = gbt::elinkMappings::VFAT_TO_GBT[vfatN];
+    const uint32_t gbtN = elinkMappings::VFAT_TO_GBT[vfatN];
 
     for (unsigned char regN = 0; regN < 3; regN++) {
-        const uint16_t regAddress = gbt::elinkMappings::ELINK_TO_REGISTERS[gbt::elinkMappings::VFAT_TO_ELINK[vfatN]][regN];
+        const uint16_t regAddress = elinkMappings::ELINK_TO_REGISTERS[elinkMappings::VFAT_TO_ELINK[vfatN]][regN];
 
-        if (writeGBTRegLocal(la, ohN, gbtN, regAddress, phase))
+        if (writeGBTRegLocal(ohN, gbtN, regAddress, phase))
             return true;
     }
 
     return false;
+
 } //End writeGBTPhaseLocal
 
-bool writeGBTRegLocal(localArgs *la, const uint32_t ohN, const uint32_t gbtN, const uint16_t address, const uint8_t value)
+bool writeGBTRegLocal(const uint32_t ohN, const uint32_t gbtN, const uint16_t address, const uint8_t value)
 {
     // Check that the GBT exists
-    if (gbtN >= gbt::GBTS_PER_OH)
-        EMIT_RPC_ERROR(la->response, stdsprintf("The gbtN parameter supplied (%u) is larger than the number of GBT's per OH (%u).", gbtN, gbt::GBTS_PER_OH), true);
+    if (gbtN >= GBTS_PER_OH)
+        EMIT_RPC_ERROR(stdsprintf("The gbtN parameter supplied (%u) is larger than the number of GBT's per OH (%u).", gbtN, GBTS_PER_OH), true);
 
     // Check that the address is writable
-    if (address >= gbt::CONFIG_SIZE)
-        EMIT_RPC_ERROR(la->response, stdsprintf("GBT has %hu writable addresses while the provided address is %hu.", gbt::CONFIG_SIZE-1, address), true);
+    if (address >= CONFIG_SIZE)
+        EMIT_RPC_ERROR(stdsprintf("GBT has %hu writable addresses while the provided address is %hu.", CONFIG_SIZE-1, address), true);
 
     // GBT registers are 8 bits long
-    writeReg(la, "GEM_AMC.SLOW_CONTROL.IC.READ_WRITE_LENGTH", 1);
+    writeReg("GEM_AMC.SLOW_CONTROL.IC.READ_WRITE_LENGTH", 1);
 
     // Select the link number
-    const uint32_t linkN = ohN*gbt::GBTS_PER_OH + gbtN;
-    writeReg(la, "GEM_AMC.SLOW_CONTROL.IC.GBTX_LINK_SELECT", linkN);
+    const uint32_t linkN = ohN*GBTS_PER_OH + gbtN;
+    writeReg("GEM_AMC.SLOW_CONTROL.IC.GBTX_LINK_SELECT", linkN);
 
     // Write to the register
-    writeReg(la, "GEM_AMC.SLOW_CONTROL.IC.ADDRESS", address);
-    writeReg(la, "GEM_AMC.SLOW_CONTROL.IC.WRITE_DATA", value);
-    writeReg(la, "GEM_AMC.SLOW_CONTROL.IC.EXECUTE_WRITE", 1);
+    writeReg("GEM_AMC.SLOW_CONTROL.IC.ADDRESS", address);
+    writeReg("GEM_AMC.SLOW_CONTROL.IC.WRITE_DATA", value);
+    writeReg("GEM_AMC.SLOW_CONTROL.IC.EXECUTE_WRITE", 1);
 
     return false;
 } //End writeGBTRegLocal(...)
```

Comment:
let's also remove all of these location awareness "end" comments

Reply:
What do you mean by location awareness here? Is it just anything at all using the ``writeReg``? Is it things specifically writing the ``address`` to the registry? if that's the case is there anything else location aware I should remove?

Reply:
i mean, the comment tries to tell someone reading the code where this brace belongs, while most editors can tell the developer this without the need for the comment, so this is a general cleanup that i'd like to take care of while this migration is happening
also
* consistent formatting of braces
* consistent usage of preincrement (unless explicitly using the post-increment behaviour differences)
* consistent removal of trailing whitespace
* consistent whitespace usage
* cleanup of extraneous comments

Reply:
Oh! I see now. Sorry, I thought you might have been using some sort of jargon to refer to the code itself but now I understand that you were referring to the comments. Thanks.